hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -416,7 +417,7 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
                 self.log_with_clock(
                     logging.INFO,
                     f"({market_pair.taker.trading_pair}) Taker buy order {order_id} for "
-                    f"({market_order_record.amount} {market_order_record.base_asset} has been completely filled."
+                    f"({market_order_record.quantity} {market_order_record.base_asset} has been completely filled."
```

Comment:
The old log code was correct. `MarketOrder` type has `.amount`, not `.quantity`.